src/main/java/com/amadeus/ereputation/HotelSentiments.java

```diff
@@ -0,0 +1,21 @@
+package com.amadeus.ereputation;
+
+import com.amadeus.Amadeus;
+import com.amadeus.Params;
+import com.amadeus.Response;
+import com.amadeus.exceptions.ResponseException;
+import com.amadeus.resources.HotelSentiment;
+import com.amadeus.resources.Resource;
+
+public class HotelSentiments {
+  private Amadeus client;
+
+  public HotelSentiments(Amadeus client) {
+    this.client = client;
+  }
+
+  public HotelSentiment[] get(Params params) throws ResponseException {
+    Response response = client.get("/v2/e-reputation/hotel-sentiments", params);
+    return (HotelSentiment[]) Resource.fromArray(response, HotelSentiment[].class);
+  }
```

Comment:
```
  /**
   * Convenience method for calling <code>get</code> without any parameters.
   * @see HotelSentiment#get()
   */
  public HotelSentiment[] get() throws ResponseException {
    return get(null);
  }
```

Reply:
The above method has mandatory parameters so i ignored the common `get` without params method in order to be less confusing for the developer